src/app/create-event/page.tsx

```diff
@@ -99,7 +57,7 @@ export default function CreateEvent() {
       mode === 'weekly'
         ? daysOfWeekJSON
         : JSON.parse('{"days": ["2024-01-01"]}'), // filler for specific mode now because no calendar yet
-      '9e33186f-95db-4385-a974-ee38c8e07547',
+      'testUsername',
```

Comment:
Is this needed? Can it be skipped instead of passing in a sentinel value all the time?

Reply:
Removed sentinel value.